readme.md

```diff
@@ -180,6 +180,7 @@ And [many more…](source/content.css)
 - [Hide comments in fewer clicks.](https://user-images.githubusercontent.com/1402241/43039221-1ddc91f6-8d29-11e8-9ed4-93459191a510.gif)
 - [Filter PRs that you haven't submitted a review for.](https://user-images.githubusercontent.com/9264728/42569205-0c32a376-8510-11e8-9406-b3ddcf6087ef.png)
 - Use <kbd>←</kbd> and <kbd>→</kbd> to navigate through pages with pagination.
+- [Use <kbd>↑</kbd> and <kbd>↓</kbd> to go to the last and first item in select lists.](https://github.com/sindresorhus/refined-github/issues/1450)
```

Comment:
`to go to` => `to also go to` to make it clear that it doesn't just work with the last and first item.